mobile/lib/features/wallet/wallet_screen.dart

```diff
@@ -105,7 +105,7 @@ class WalletScreen extends StatelessWidget {
                               size: 14,
                             ),
                             SizedBox(
-                              width: 10,
+                              width: 7,
```

Comment:
man, today you have the eastereggs hidden!

3  `7` changed in this file = `3 * 7` = `21` = `10101`